lxd/storage_volumes_snapshot.go

```diff
@@ -229,8 +229,8 @@ func storagePoolVolumeSnapshotsTypePost(d *Daemon, r *http.Request) response.Res
 		return pool.CreateCustomVolumeSnapshot(projectName, volumeName, req.Name, expiry, op)
 	}
 
-	resources := map[string][]string{}
-	resources["storage_volumes"] = []string{volumeName}
+	resources := map[string][]api.URL{}
+	resources["storage_volumes"] = []api.URL{*api.NewURL().Path(version.APIVersion, "storage-pools", poolName, "volumes", volumeTypeName, volumeName, "snapshots")}
```

Comment:
This is an incorrect URL for the "storage_volumes" category, we should retain the URL to the storage volume itself here, and then add a new resource category for "storage_volume_snapshots" pointing to the snapshot resource URL (including the snapshot name).